sample/kotlin/build.gradle.kts

```diff
@@ -112,7 +112,7 @@ android {
 dependencies {
 
     implementation(project(":dd-sdk-android"))
-    implementation(project(":library:dd-sdk-android-session-replay"))
+    implementation(project(":library:dd-sdk-android-session-replay-material"))
```

Comment:
shouldn't we have both? it will make more clear which is the core and which one is just the extension.